src/components/Card/Card.tsx

```diff
@@ -21,6 +21,9 @@ const Card = ({ title, image, id, sentiment, updateUserSentiment }: CardProps):
             <img className="article-image cy-article-image" src={image} alt={title} />
             <h2 className="article-title cy-article-title">{title}</h2>
           </Link>
+          {sentiment >= 7 && <h3 className="article-sentiment green">Sentiment rating: {sentiment}</h3>}
+          {sentiment < 4 && <h3 className="article-sentiment red">Sentiment rating: {sentiment}</h3>}
```

Comment:
This probably should be `<= 4`, otherwise a sentiment of 4 won't be covered by the ranges.
```suggestion
          {sentiment <= 4 && <h3 className="article-sentiment red">Sentiment rating: {sentiment}</h3>}
```